src/views/count.vue

```diff
@@ -115,6 +161,35 @@
         } else {
           showToast(translate("Enter the stock count for the product"))
         }
+      },
+      async getFacilityLocations() {
+        let resp;
+        try {
+          const params = {
+            "inputFields": {
+              facilityId: this.facility.facilityId,
+              productId: this.product.productId,
+            },
+            "entityName": "ProductAndFacilityLocation",
+            "fieldList": ["areaId", "aisleId", "sectionId", "levelId", "positionId", "locationSeqId"]
+          }
+          resp = await ProductService.getFacilityLocations(params);
+          if(resp.status === 200 && resp.data.count && resp.data.count > 0 && !hasError(resp)) {
+            this.facilityLocations = resp.data.docs;
+
+            this.facilityLocations = this.facilityLocations.map((location: any) => {
+              return {
+                locationSeqId: location.locationSeqId,
+                locationPath: location.areaId + location.aisleId + location.sectionId + location.levelId + location.positionId
+              }
+            })
+            this.location = this.facilityLocations[0].locationPath;
+            this.product.locationId = this.facilityLocations[0].locationSeqId;
```

Comment:
I think we should not set locationId for product as we already do it for confirmation. Let me know I miss something here

Reply:
Sir, in line no. 187 we are trying to set locationId for product on initial load with 0th facility locationSeqId because by default a product does not have locationId. But we can remove this line of code if you think that we should not initially assign locationId to the product **WDYT**.

Reply:
We could skip for now